string name
# for now the frame has to be map
# string frame_id
sensor_msgs/Image image
# grid spacing
float64 resolution
# multiply the height value by this
float64 height_scale
bool flip_quad_edges
